pandas/core/arrays/categorical.py

```diff
@@ -2316,7 +2316,7 @@ def _concat_same_type(self, to_concat):
 
         return union_categoricals(to_concat)
 
-    def isin(self, values):
+    def isin(self, values) -> np.ndarray:
```

Comment:
so here we *know* that this returns a bool dtyped ndarray, is it worth *somehow* annotating this as such?

Reply:
IIRC there has been some discussion in numpy about making it viable to annotate something like `np.ndarray[bool]`, but it is still hypothetical (cc @shoyer?)